package.json

```diff
@@ -8,6 +8,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "aws-sdk": "^2.1457.0",
```

Comment:
Is this dependency needed? Since we are doing S3 related operations in the submodule.

Reply:
This library requires to run the S3 operation in the submodule.